### ISSUE TITLE
no longer writing scores to DB?

### ISSUE THREAD

**Issue by kuaka** [FAI-CIVL/FAI-Airscore#179]
are scores no longer written to the DB?
![image](https://user-images.githubusercontent.com/19358239/99897055-2a1eb100-2cfb-11eb-9ca1-265bffe0d60f.png)

I'm guessing that this stopped when result files started getting made?
would be nice to have them. As it stands it is misleading having the columns if they are not used.
```
        task_obj = Task.read(task_id)
        task_obj.get_results()
```

then 
task_obj.arrival_score (or any other score) is 0

**Comment by biuti** [comment]
Yes, we are not using them. To me it would be misleading to have any score there, as final results could be different from the stored ones, unless we store only when and if a result is flagged final. That could be an option, otherwise I would rather delete the columns.
I'm thinking if we need a new column, I would use resultfile.status as flag, like 'prov.', 'test', 'final', and a new column comment that substitutes / adds to the flag in frontend, if not null.
